onelogin_aws_cli/configuration.py

```diff
@@ -68,8 +78,10 @@ def section(self, section_name: str):
         :param section_name: Name of the section in the config file
         :return:
         """
-        if not self.has_section(section_name):
-            self.add_section(section_name)
+        section_missing = not self.has_section(section_name)
+        not_default = self.default_section != section_name
```

Comment:
Could probably use some coverage for this case, but your other PR will probably cover it just by calling login without -C

Reply:
Yes, I was thinking that when the test cases passed after I modified it